fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -1270,7 +1271,7 @@ private void addListPartitionLog(Database db, OlapTable olapTable, List<Partitio
             boolean isTempPartition = addPartitionClause.isTempPartition();
             if (existPartitionNameSet.contains(partition.getName())) {
                 LOG.info("add partition[{}] which already exists", partition.getName());
-                return;
```

Comment:
is it necessary to add the log above?

Reply:
Yes, there are some concurrency issues that still need this log to troubleshoot.